sdk/storage/inc/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -2762,21 +2761,33 @@ namespace Azure { namespace Storage { namespace DataLake {
           {
             result.AcceptRanges = response.GetHeaders().at(Details::c_HeaderAcceptRanges);
           }
-          if (response.GetHeaders().find("Cache-Control") != response.GetHeaders().end())
+          if (response.GetHeaders().find(
+                  "<built-in method lower of str object at 0x000001B7FAFE7BF0>")
```

Comment:
what?

Reply:
Let's just pretend you did not see this.